naresh/src/main/java/com/lftechnology/phpjava/assignmenttwo/SalutationSplitter.java

```diff
@@ -1,21 +1,45 @@
 package com.lftechnology.phpjava.assignmenttwo;
 
+import com.lftechnology.phpjava.menu.MenuReader;
+import com.lftechnology.phpjava.menu.ProgramMenu;
+
+import java.util.Scanner;
+
 /**
  * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
  */
-public class SalutationSplitter {
-    public static String name = "";
-    public static String salutaion = "";
-
-    public static void main(String[] args) {
-        for (int i = 0; i < args.length; i++) {
-            if (i == 0) {
-                name = args[0];
-            } else {
-                salutaion = salutaion + " " + args[i];
+public class SalutationSplitter implements MenuReader {
+    public void salutationSplitter(String input) {
+        String name = "";
+        String[] nameAndSalutation = input.split(" ");
+        String salutaion = nameAndSalutation[0];
+        String space = "";
+        for (int i = 0; i < nameAndSalutation.length; i++) {
+            if (i > 0) {
+                name += space + nameAndSalutation[i];
+                space = " ";
             }
         }
-        System.out.println(name);
-        System.out.println(salutaion);
+        System.out.println("\n\nSalutation : " + salutaion);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)